readme_bridge.md

```diff
@@ -71,6 +71,112 @@ Then you can start the server:
+
+#### Request
+
+Check example request below (remove comments before submitting it to the `bridge` server):
+
+```json
+{
+  // Transaction source account
+  "source": "GBDCOZD7CHY26KS6ABEZPIJAMS2G7GP3YSTJ6DIRIQ6YUU77ZAPI2LVT",
+  // Sequence number
+  "sequence_number": "123",
+  // List of operations in this transaction
+  "operations": [
+    // First operation
+    {
+      // Operation type
+      "type": "create_account",
+      // Operation body
+      "body": {
+        // Don't send source field if operation source account is equal to transaction source account
+        "source": "GBDCOZD7CHY26KS6ABEZPIJAMS2G7GP3YSTJ6DIRIQ6YUU77ZAPI2LVT",
+        "destination": "GBDCOZD7CHY26KS6ABEZPIJAMS2G7GP3YSTJ6DIRIQ6YUU77ZAPI2LVT",
+        "starting_balance": "50"
+      }
+    },
+    // Second operation
+    {
+      "type": "payment",
+      "body": {
+        "source": "GBDCOZD7CHY26KS6ABEZPIJAMS2G7GP3YSTJ6DIRIQ6YUU77ZAPI2LVT",
+        "destination": "GBDCOZD7CHY26KS6ABEZPIJAMS2G7GP3YSTJ6DIRIQ6YUU77ZAPI2LVT",
+        "amount": "1050",
+        "asset": {
+          "code": "USD",
+          "issuer": "GBDCOZD7CHY26KS6ABEZPIJAMS2G7GP3YSTJ6DIRIQ6YUU77ZAPI2LVT"
+        }
+      }
+    },
+    {
+      "type": "path_payment",
+      "body": {
+        "source": "GBDCOZD7CHY26KS6ABEZPIJAMS2G7GP3YSTJ6DIRIQ6YUU77ZAPI2LVT",
+        "destination": "GBDCOZD7CHY26KS6ABEZPIJAMS2G7GP3YSTJ6DIRIQ6YUU77ZAPI2LVT",
+        "destination_amount": "10",
+        "send_max": "1050",
+        "send_asset": {
+          "code": "USD",
+          "issuer": "GBDCOZD7CHY26KS6ABEZPIJAMS2G7GP3YSTJ6DIRIQ6YUU77ZAPI2LVT"
+        },
+        "destination_asset": {
+          "code": "EUR",
+          "issuer": "GBDCOZD7CHY26KS6ABEZPIJAMS2G7GP3YSTJ6DIRIQ6YUU77ZAPI2LVT"
+        },
+        "path": [
+	        {}, // Native asset
+      		{
+              "code": "ZAR",
+              "issuer": "GBNIVKJTD2SMAXB5ALPBZ7CHRYYLCO5XSH55H6TI3Z37P7SCRXQVESG2"
+            }
+        ]
+      }
+    },
+    {
+      "type": "change_trust",
+      "body": {
+        "source": "GBDCOZD7CHY26KS6ABEZPIJAMS2G7GP3YSTJ6DIRIQ6YUU77ZAPI2LVT",
+        "asset": {
+          "code": "USD",
+          "issuer": "GBDCOZD7CHY26KS6ABEZPIJAMS2G7GP3YSTJ6DIRIQ6YUU77ZAPI2LVT"
+        }
+      }
+    },
+    {
+      "type": "allow_trust",
+      "body": {
+        "source": "GBDCOZD7CHY26KS6ABEZPIJAMS2G7GP3YSTJ6DIRIQ6YUU77ZAPI2LVT",
+        "trustor": "GBDCOZD7CHY26KS6ABEZPIJAMS2G7GP3YSTJ6DIRIQ6YUU77ZAPI2LVT",
+        "asset_code": "USD",
+        "authorize": true
+      }
+    }
+  ],
+  // Array of signers
+  "signers": ["SDOTALIMPAM2IV65IOZA7KZL7XWZI5BODFXTRVLIHLQZQCKK57PH5F3H"]
+}
+```
+
+Assets are represented by a JSON object with two fields: `code` and `issuer`. Empty JSON object represents [native asset](https://www.stellar.org/developers/learn/concepts/assets.html#lumens-xlm-).
+
+#### Response
```

Comment:
I think it should just submit the tx. If they are using the bridge they aren't dealing directly with horizon.